packages/browserify/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lavamoat-browserify",
-  "version": "16.0.0",
```

Comment:
16.0.0 was a failing attempt, we should deprecate this version from npm registry